src/daemon/daemon.cpp

```diff
@@ -457,6 +457,19 @@ auto try_mem_size(const std::string& val) -> std::optional<mp::MemorySize>
     }
 }
 
+std::string get_bridged_interface_name()
+{
+    const auto bridged_id = MP_SETTINGS.get(mp::bridged_interface_key);
+
+    if (bridged_id == "")
+    {
+        throw std::runtime_error(fmt::format("You have to `multipass set {}=<name>` to use the `--bridged` shortcut.",
```

Comment:
`--bridged` is not applicable in the case of settings, so the message needs to be generalized. Perhaps just:

```suggestion
        throw std::runtime_error(fmt::format("You have to `multipass set {}=<name>` to use the "bridged" shortcut.",
```

Reply:
Right, changed.